.storybook/addons.js

```diff
@@ -0,0 +1,4 @@
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
+import 'storybook-addon-jsx/register';
```

Comment:
See the no new line icon. Would be good to use same `.editorconfig` as blockstack js (will also prevent trialing whitespace)